src/main/java/net/swofty/event/actions/player/data/ActionPlayerDataSave.java

```diff
@@ -38,7 +34,7 @@ public void run(Event event) {
         player.getDataHandler().runOnSave(player);
 
         /*
-        Save the data into the DB
+         * Save the data into the DB
```

Comment:
This comment was intended to be of type `/*`, only the `/**` comments as seen in the SkyBlock.java class have asterisks in the middle